tests/Unit/Elliptic/Test_FirstOrderOperator.cpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Elliptic/FirstOrderOperator.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+#include "tests/Utilities/MakeWithRandomValues.hpp"
+
+namespace {
+
+template <typename Tag>
+struct Step : db::SimpleTag, db::PrefixTag {
+  using type = typename Tag::type;
+  static std::string name() noexcept { return "Step"; }
+  using tag = Tag;
+};
+
+struct SomeField : db::SimpleTag {
+  using type = Scalar<DataVector>;
+  static std::string name() noexcept { return "SomeField"; }
```

Comment:
Do you have to specify the `name()` if it's the same as the struct's name?

Reply:
DataBox tags names don't default to the class names (yet).

Reply:
Yes they do.  That feature was added with PDALs (#1535).  (Some things still try to call `Tag::name()` directly instead of `db::tag_name<Tag>()`, but those are easy-to-fix bugs.)

Reply:
You're right of course. I didn't realize `db::tag_name` had that functionality 👍